x-pack/plugins/monitoring/public/application/pages/page_template.tsx

```diff
@@ -103,31 +111,37 @@ export const PageTemplate: React.FC<PageTemplateProps> = ({
   };
 
   return (
-    <div className="app-container" data-test-subj="monitoringAppContainer">
-      <ActionMenu>
-        <AlertsDropdown />
-      </ActionMenu>
-      <MonitoringToolbar pageTitle={pageTitle} onRefresh={onRefresh} />
-      {tabs && (
-        <EuiTabs>
-          {tabs.map((item, idx) => {
-            return (
-              <EuiTab
-                key={idx}
-                disabled={isDisabledTab(product)}
-                title={item.label}
-                data-test-subj={item.testSubj}
-                href={createHref(item.route)}
-                isSelected={isTabSelected(item.route)}
-              >
-                {item.label}
-              </EuiTab>
-            );
-          })}
-        </EuiTabs>
-      )}
-      <div>{renderContent()}</div>
-    </div>
+    <EuiPage data-test-subj="monitoringAppContainer">
+      <EuiPageBody>
+        <EuiPageContent>
+          <ActionMenu>
+            <div id="setupModeNav">{/* HERE GOES THE SETUP BUTTON */}</div>
```

Comment:
@miltonhultgren I moved the Setup Mode button here, but I wasn't able to get it to appear in the top navigation. Let me know if I can help at all. Thanks!

Reply:
This is what is happening:

1. The page loads
2. The setup mode button is rendered into the `div` (in the toolbar)
3. Something causes the page to re-render which clear the Setup mode button and puts back the `div` (since that is what we asked this component to do, render a `div`)

The solutions for this are:

1. Call the Setup mode rendering code on each page render (feels bad)
2. Stop the page from re-rendering (impossible and would break the page)
3. Make the Setup mode button render naturally as part of the page rendering (if possible) 

I'm going down path 3 and will see where the dragons are :)